<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>829</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>791</width>
     <height>431</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>有线网络</string>
    </attribute>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>481</width>
       <height>272</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="labelIPaddress">
        <property name="text">
         <string>IP     地址</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditIPaddress"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelNetmask">
        <property name="text">
         <string>子网掩码</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEditNetmask"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelGateway">
        <property name="text">
         <string>默认网关</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEditGateway"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="labelDNS">
        <property name="text">
         <string>DNS        </string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="lineEditDNS"/>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QComboBox" name="comboBoxIP">
        <item>
         <property name="text">
          <string>自动获得IP地址</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>使用下面的IP地址</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QComboBox" name="comboBoxDNS">
        <item>
         <property name="text">
          <string>自动获得DNS服务器地址</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>使用下面的DNS服务器地址</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="Save">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>290</y>
       <width>81</width>
       <height>44</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>28</pointsize>
      </font>
     </property>
     <property name="text">
      <string>设置</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Wifi配置</string>
    </attribute>
    <widget class="QPushButton" name="WifiScan">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>270</y>
       <width>151</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>扫描</string>
     </property>
    </widget>
    <widget class="QPushButton" name="WifiSetting">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>270</y>
       <width>151</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>设置</string>
     </property>
    </widget>
    <widget class="QComboBox" name="WifiSSID">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>180</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="WIFIPASS">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>230</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Wifi_switch">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>110</y>
       <width>311</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Wifi开</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSocketWifiStatus">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>10</y>
       <width>261</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>wifi连接状态</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Wifi 开关状态</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>10</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>扫描已配置wifi</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSocketWifiStatus_2">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>70</y>
       <width>261</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>wifi密码保存</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSocketWifiStatus_3">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>130</y>
       <width>261</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>json To conf</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Bluetooth</string>
    </attribute>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>141</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>scan</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>SystemIO</string>
    </attribute>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>401</width>
       <height>141</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="3">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>查找</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Progress  </string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>杀死进程</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>PID</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item row="2" column="2">
       <widget class="QComboBox" name="comboBox_2"/>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>20</y>
       <width>221</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>Sys屏幕校准</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>100</y>
       <width>221</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>QT屏幕校准</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
